testsuite/tests/apicast/parameters/test_date_logging.py

```diff
@@ -0,0 +1,40 @@
+"""
+Test for logging policy with custom date format
+"""
+from datetime import datetime, timezone
+
+import pytest
+
+from testsuite import rawobj
+from testsuite.capabilities import Capability
+
+pytestmark = [
+    pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-6594"),
+    pytest.mark.required_capabilities(Capability.LOGS),
+]
+
+
+@pytest.fixture(scope="module")
+def policy_settings():
+    """Customize the access logs format"""
```

Comment:
`policy_settings` works nicely with predefined customized `service`. if that doesn't exist (to share simpler definition between other tests) neither `policy_settings` makes sense.

1. Either place policy definition directly to `service` fixture
2. or put the `service` fixture honoring `policy_settings` to conftest here or upstrairs